confidential-data-hub/storage/src/volume_type/blockdevice/luks/mod.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2024 Intel
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+use super::Encryptor;
+use super::{get_plaintext_key, BlockDeviceError, BlockDeviceParameters, Result};
+use async_trait::async_trait;
+use base64::Engine;
+use log::error;
+use rand::{distributions::Alphanumeric, Rng};
+use tokio::{
+    fs,
+    io::{AsyncReadExt, AsyncWriteExt},
+    process::Command,
+};
+
+/// LUKS encrypt storage binary
+const LUKS_ENCRYPT_STORAGE_BIN: &str = "/usr/local/bin/luks-encrypt-storage";
+
+async fn random_encrypt_key() -> anyhow::Result<String> {
+    let mut buffer = vec![0u8; 4096];
+    rand::thread_rng().fill(&mut buffer[..]);
+    Ok(base64::engine::general_purpose::STANDARD.encode(&buffer))
```

Comment:
This function does not return `Err`, thus we could directly use `async fn random_encrypt_key() -> String`?

Reply:
Done.